features/src/rapids-build-utils/opt/rapids-build-utils/manifest.yaml

```diff
@@ -102,7 +102,7 @@ repos:
 
 - name: raft
   path: raft
-  git: {<<: *git_defaults, repo: raft}
+  git: {host: github, upstream: jameslamb, repo: raft, tag: wheel-validation}
```

Comment:
temporarily adding this to test the fixes for this RAFT issue: https://github.com/rapidsai/raft/pull/2490#issuecomment-2474990415